.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+ workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.5.3
+
+    - name: Update Rubygems
+      run: gem update --system 3.1.0
+
+    - name: Clone repository
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.GITHUB_REF }}
+        # Needed to fetch all the tags
+        fetch-depth: 0
+
+    - name: Tag repository
+      run: |
+        module_version=$(cat metadata.json | jq '.version' -r)
+        tag="v${module_version}"
+        if git rev-parse "${tag}" >/dev/null 2>&1; then
+          echo "Module's already been tagged; skipping to the Forge publish step";
+        else
+          git tag ${tag}
+          git push -f --tags
+        fi
+
+    - name: Publish module to the forge
+      env:
+        BLACKSMITH_FORGE_TOKEN: ${{ secrets.FORGE_API_KEY }}
```

Comment:
@gsparks Think this will be needed in splunk_hec, b/c I copy-pasted your job and had to tweak some things to get it working.